docs/asciidoc/static/include/pluginbody.asciidoc

```diff
@@ -0,0 +1,1021 @@
+:branch:          1.5
+:ls_version:		  1.5.0.beta1
+
+= How to write your own Logstash {plugintype} plugin
+
+This HOWTO will walk you through the process of developing a new {plugintype}
+for Logstash. The workflow has changed as of version 1.5 of Logstash, so you'll
+want to follow these new and improved steps to contribute a new plugin!  Since
+Logstash 1.5, all plugins are self-contained ruby gems.  In all previous
+versions, plugins were part of the core Logstash distribution.  This change
+makes it possible to develop plugins independent of the Logstash core, and
+release updates independently as well!
+
+NOTE: Logstash is written in Ruby.  If you are unfamiliar with the Ruby programming
+language, an excellent quick-start guide is at
+https://www.ruby-lang.org/en/documentation/quickstart/.
+
+== Get started
+
+{getstarted}
+
+=== Create a new GitHub repo
```

Comment:
"Create a new" is redundant...you wouldn't expect to create an old one :-) I would flip this around to say "Create a GitHub repo for the new plugin"